lib/service-manager.js

```diff
@@ -322,6 +322,10 @@ prototype.getInstanceEnv = function(instanceId, callback) {
         );
 
       var env = getServiceEnv(service);
+
+      // Set override env so strong-trace returns the instanceId as the hostname
+      env.STRONGLOOP_HOSTNAME_OVERRIDE = instanceId;
```

Comment:
Should this include the service id as well? Like `fmt('%s-%s', serviceId, instanceId);` or something..